### PR TITLE
[Excel] (Custom Functions) Add note on Simple CORS

### PR DESCRIPTION
**Pull request by keyur32**

### CODE REVIEW

**Comment by o365devx** [comment]
OPS Build status updates of commit _[a4fc1aa](https://github.com/OfficeDev/office-js-docs-pr/pull/727/commits/a4fc1aa42e5eb9bf888384e8e58c4ab23717fe85)_: 

:clock10: **Preparing**: average preparing time is _44 sec(s)_

**Comment by o365devx** [comment]
OPS Build status updates of commit _[a4fc1aa](https://github.com/OfficeDev/office-js-docs-pr/pull/727/commits/a4fc1aa42e5eb9bf888384e8e58c4ab23717fe85)_: 

:clock10: **Incremental building**: average incremental building time is _29 sec(s)_

**Comment by o365devx** [comment]
OPS Build status updates of commit _[a4fc1aa](https://github.com/OfficeDev/office-js-docs-pr/pull/727/commits/a4fc1aa42e5eb9bf888384e8e58c4ab23717fe85)_: 

### :white_check_mark: Validation status: passed


File | Status | Preview URL | Details
---- | ------ | ----------- | -------
[docs/excel/custom-functions-runtime.md](https://github.com/OfficeDev/office-js-docs-pr/blob/keyur32-patch-1/docs/excel/custom-functions-runtime.md) | :white_check_mark:Succeeded | |

For more details, please refer to the [build report](https://opbuildstorageprod.blob.core.windows.net/report/2019%5C2%5C1%5C57856ef7-e8cf-54ca-0d12-df5f42d1e8bb%5CPullRequest%5C201902011951562191-727%5Cworkflow_report.html?sv=2015-02-21&sr=b&sig=5wIyEr6oNM1rhXHZ2TlmJJSmOnNsa6YIbnSjJDTywqY%3D&st=2019-02-01T19%3A48%3A13Z&se=2019-03-04T19%3A53%3A13Z&sp=r).

**Note:** If you changed an existing file name or deleted a file, broken links in other files to the deleted or renamed file are listed only in the full build report.

**Comment by keyur32** [comment]
@mscharlock , @davidchesnut  - please review. if you agree, can we add a note to our tutorial as well?

**Comment by keyur32** [comment]
Also, could we add a note to specify only these types are supported for the XHR requests:

application/x-www-form-urlencoded
multipart/form-data
text/plain

**Comment by kbrandl** [comment]
(Re-titled this PR using our standard convention.)

File: docs/excel/custom-functions-runtime.md [hunk 1]
```diff
@@ -39,6 +39,9 @@ function sendWebRequest(thermometerID, data) {
         if (this.readyState == 4 && this.status == 200) {
            data.temperature = JSON.parse(xhttp.responseText).temperature
         };
+        
+        //set Content-Type to application/text. Application/json is not currently supported with Simple CORS
```
Comment (Reezaali) [inline docs/excel/custom-functions-runtime.md]: @keyur32  can you change to the following please: 

// Set ...

Comment (mscharlock) [inline docs/excel/custom-functions-runtime.md]:
```suggestion
        //Set Content-Type to application/text. Application/json is not currently supported with Simple CORS.
```

**Comment by o365devx** [comment]
OPS Build status updates of commit _[487624e](https://github.com/OfficeDev/office-js-docs-pr/pull/727/commits/487624eaab21a3cec87c8161c8d4bbd4f97c2935)_: 

:clock10: **Preparing**: average preparing time is _44 sec(s)_

**Comment by o365devx** [comment]
OPS Build status updates of commit _[487624e](https://github.com/OfficeDev/office-js-docs-pr/pull/727/commits/487624eaab21a3cec87c8161c8d4bbd4f97c2935)_: 

### :white_check_mark: Validation status: passed


File | Status | Preview URL | Details
---- | ------ | ----------- | -------
[docs/excel/custom-functions-runtime.md](https://github.com/OfficeDev/office-js-docs-pr/blob/keyur32-patch-1/docs/excel/custom-functions-runtime.md) | :white_check_mark:Succeeded | |

For more details, please refer to the [build report](https://opbuildstorageprod.blob.core.windows.net/report/2019%5C2%5C4%5C57856ef7-e8cf-54ca-0d12-df5f42d1e8bb%5CPullRequest%5C201902041901275613-727%5Cworkflow_report.html?sv=2015-02-21&sr=b&sig=UB1c5vBKB%2FYQYIJvVuQ%2Fm2kgEmZ8Hdx%2FE9xe0xVo2FI%3D&st=2019-02-04T18%3A57%3A38Z&se=2019-03-07T19%3A02%3A38Z&sp=r).

**Note:** If you changed an existing file name or deleted a file, broken links in other files to the deleted or renamed file are listed only in the full build report.

**Comment by o365devx** [comment]
OPS Build status updates of commit _[94b463c](https://github.com/OfficeDev/office-js-docs-pr/pull/727/commits/94b463cc6bd5792663fe4b60736b5baea3d0bbb0)_: 

:clock10: **Preparing**: average preparing time is _44 sec(s)_

**Comment by o365devx** [comment]
OPS Build status updates of commit _[94b463c](https://github.com/OfficeDev/office-js-docs-pr/pull/727/commits/94b463cc6bd5792663fe4b60736b5baea3d0bbb0)_: 

:clock10: **Incremental building**: average incremental building time is _29 sec(s)_

**Comment by o365devx** [comment]
OPS Build status updates of commit _[94b463c](https://github.com/OfficeDev/office-js-docs-pr/pull/727/commits/94b463cc6bd5792663fe4b60736b5baea3d0bbb0)_: 

### :white_check_mark: Validation status: passed


File | Status | Preview URL | Details
---- | ------ | ----------- | -------
[docs/excel/custom-functions-runtime.md](https://github.com/OfficeDev/office-js-docs-pr/blob/keyur32-patch-1/docs/excel/custom-functions-runtime.md) | :white_check_mark:Succeeded | |

For more details, please refer to the [build report](https://opbuildstorageprod.blob.core.windows.net/report/2019%5C2%5C6%5C57856ef7-e8cf-54ca-0d12-df5f42d1e8bb%5CPullRequest%5C201902062226022844-727%5Cworkflow_report.html?sv=2015-02-21&sr=b&sig=3V1xg3fHav5ILaRdfAzVolLjA2AM1ebriDRiGtW%2BGrE%3D&st=2019-02-06T22%3A22%3A16Z&se=2019-03-09T22%3A27%3A16Z&sp=r).

**Note:** If you changed an existing file name or deleted a file, broken links in other files to the deleted or renamed file are listed only in the full build report.

**Pull request merged**